Nodes/Maths Functions/Math Expression.clbnode

```diff
@@ -0,0 +1,6 @@
+INPUT MathExpression `Expr`;
+
+float main(float x)
+{
+    return math_expression_result(x);
```

Comment:
Once you use the `name` parameter in `gen_input_shader_code__impl()` you will be able to use `` `Expr` `` instead of hardcoding `math_expression_result`